hydra-node/json-schemas/api.yaml

```
@@ -791,13 +791,14 @@ components:
       description: |
           Emitted by the server after drafting a commit transaction with the user provided utxos. Transaction returned to the user is in it's cbor representation encoded as Base16.
       payload:
-        type: object
-        required:
-          - commitTx
+        $ref: "#/components/schemas/RawTransaction"
         additionalProperties: false
```

Must: fix the schema

This is wrong, it's a `TextEnvelopeTransaction`

```suggestion
        $ref: "#/components/schemas/TextEnvelopeTransaction"
```

---

Should: also fix the TextEnvelopeTransaction schema which uses still the old `type`

https://github.com/input-output-hk/hydra/blob/e0f011829e60ca409e9c59e23bc0d2f5e0d4f02b/hydra-node/json-schemas/api.yaml#L1374 should say `Tx BabbageEra` as we usually produce and consume babbage era transactions.

---

Done in https://github.com/input-output-hk/hydra/commit/6dc6a9cecf7036cad47c68e8815ef48925e4da34